### PR TITLE
docs(looker): add guide to setup `dagster-looker` with the Looker API

### PR DESCRIPTION
**Pull request by rexledesma**
## Summary & Motivation
As the title.

## How I Tested These Changes
local

## Changelog
NOCHANGELOG

### CODE REVIEW

**Comment by rexledesma** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24694?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#24694** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24694?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#24692** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24692?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rexledesma and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/content/integrations/looker.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: "Using Dagster with the Looker API"
```
Comment (maximearmstrong) [inline docs/content/integrations/looker.mdx:2]: nit: 

```suggestion
title: "Using Dagster with Looker"
```

File: docs/content/integrations/looker.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+# Using Dagster with Looker
+
+<ExperimentalCallout />
+
+This guide provides instructions for using Dagster with Looker. Your Looker assets, such as views, explores, and dashboards, can be represented in the Dagster asset graph, allowing you to track lineage and dependencies between Looker assets and upstream data assets you are already modeling in Dagster. You can also use Dagster to orchestrate Looker PDTs, allowing you to trigger refreshes of these materialized tables on a cadence or based on upstream data changes.
+
+## What you'll learn
+
+- How to represent Looker assets in the Dagster asset graph, including lineage to other Dagster assets.
```
Comment (maximearmstrong) [inline docs/content/integrations/looker.mdx:14]: Are we covering lineage to other Dagster assets in this guide? It looks like that section might be missing.

Comment (rexledesma) [reply docs/content/integrations/looker.mdx:14]: That can be done in a separate PR

Comment (maximearmstrong) [reply docs/content/integrations/looker.mdx:14]: That makes sense. Let's just remove "including lineage to other Dagster assets" for now.

**Review by maximearmstrong — APPROVED** [review]
One comment, aside from that LGTM!

**Comment by rexledesma** [comment]
### Merge activity

* **Sep 26, 8:59 AM PDT**: @rexledesma merged this pull request with [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/24694).

**Pull request merged**